modules/Bio/EnsEMBL/Production/Pipeline/AlphaFold/MakeAlphaFoldDBProteinFeatures.pm

```diff
@@ -91,14 +91,24 @@ sub new {
 =cut
 
 sub run {
-  my ($self) = @_;
+    my ($self) = @_;
+
+    $self->{'afdb_info'} = $self->parse_afdb_file();
+    unless (ref($self->{'afdb_info'}) eq 'ARRAY') {
+        die "Missing info from AFDB file. Path: " . $self->{'alpha_path'} . ". Species: " . $self->{'species'} . "\n";
+    }
 
-  $self->{'afdb_info'} = $self->parse_afdb_file();
-  $self->{'perfect_matches'} = fetch_latest_uniprot_enst_perfect_matches($self->{'rest_server'},$self->{'cs_version'});
-  #$self->{'perfect_matches'} = $self->fetch_uniprot_ensembl_matches();
-  $self->make_protein_features();
+    $self->{'perfect_matches'} = eval{fetch_latest_uniprot_enst_perfect_matches($self->{'rest_server'}, $self->{'cs_version'})};
+    unless ($self->{'perfect_matches'}) {
+        info(sprintf("No data found for species %s in GIFTS DB using endpoint %s and assembly %s. Message:\n%s", $self->{'species'}, $self->{'rest_server'}, $self->{'cs_version'}, $@));
```

Comment:
I am not sure about this statement, is the info a method I don't know about? in logger?

Reply:
See line 55 - use Bio::EnsEMBL::Utils::Exception qw(throw info);
Should land somewhere in the eHive DB, probably same spot where the errors usually go.

Reply:
Ok thanks for the insights!